src/main/java/core/basesyntax/model/TypeActivity.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.model;
+
+public enum TypeActivity {
+    BALANCE, SUPPLY, PURCHASE, RETURN;
```

Comment:
```suggestion
    BALANCE("b"),
     SUPPLY("s"),
     ...
```
ти можеш одразу зашивати значення в enum, тоді простіше буде парсити строку в enum
https://www.baeldung.com/java-enum-values

Reply:
Супер! Дякую